frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/grammar/Machete.flex

```diff
@@ -0,0 +1,58 @@
+// Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
```

Comment:
This file is representation of finite state machine that recognize this language

Reply:
For more information see https://jflex.de/manual.html#Example